app/Livewire/Server/LogDrains.php

```
@@ -177,6 +177,39 @@ public function instantSave()
+                $this->{$enabledProperty} = false;
+            } else {
+                $this->validateLogDrainSettings($type);
+                $this->isLogDrainNewRelicEnabled = $type === 'newrelic';
+                $this->isLogDrainAxiomEnabled = $type === 'axiom';
+                $this->isLogDrainCustomEnabled = $type === 'custom';
+            }
+
+            $this->syncData(true);
+
+            if ($this->server->isLogDrainEnabled()) {
+                StartLogDrain::run($this->server);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Actions\Server\StartLogDrain' in method 'toggleLogDrain'. (undefined)

(StaticAccess)